vault/logical_system_paths.go

```diff
@@ -1969,9 +2428,111 @@ func (b *SystemBackend) mountPaths() []*framework.Path {
 				},
 			},
 
-			Callbacks: map[logical.Operation]framework.OperationFunc{
-				logical.ReadOperation:   b.handleMountTuneRead,
-				logical.UpdateOperation: b.handleMountTuneWrite,
+			Operations: map[logical.Operation]framework.OperationHandler{
+				logical.ReadOperation: &framework.PathOperation{
+					Callback: b.handleMountTuneRead,
+					Responses: map[int][]framework.Response{
+						http.StatusOK: {{
+							Description: "OK",
+							Fields: map[string]*framework.FieldSchema{
+								"max_lease_ttl": {
+									Type:        framework.TypeString,
```

Comment:
```suggestion
									Type:        framework.TypeInt,
```
looks like this is cast to int
https://github.com/hashicorp/vault/blob/901876f27abc9e74904aaebaba7a31bc9cb970a6/vault/logical_system.go#L1595